cookbooks/ohai_backport/README.md

```diff
@@ -0,0 +1,13 @@
+ohai_backport Cookbook
+======================
+Stub cookbook to backport ohai plugins required for the CI to work
```

Comment:
that's not really true anymore. Might be worth calling this "fb_ci_fixes" or some such.